src/hashtables.jl

```diff
@@ -26,11 +27,13 @@ end
 function show_params_TaylorN()
     info( """`TaylorN` and `HomogeneousPolynomial` parameters:
     Maximum order       = $(_params_taylorN.maxOrder)
-    Number of variables = $(_params_taylorN.numVars)""" )
+    Number of variables = $(_params_taylorN.numVars)
+    Variable names = $(_params_taylorN.variable_names)
+    """)
     nothing
 end
 
-global const _params_taylorN = ParamsTaylorN(6,2)
+global const _params_taylorN = ParamsTaylorN(6,2, ["x", "y", "z", "t"])
```

Comment:
This indeed sounds really nice. 

How will you handle the maximum degree and, perhaps more important, the fact that we are constrained by a given number of variables? My concern is related to use it as
```julia
x, y, z = variables("x y z")
```
when `get_numvars()` returns 2. Maybe a warning or an error...